cli/azd/resources/awesome-templates.json

```diff
@@ -0,0 +1,282 @@
+[
```

Comment:
Note, this is just a placeholder till we can expose an Awesome AZD json file in the awesome-azd repo.